data_anal.py

```diff
@@ -0,0 +1,86 @@
+import glob
+import matplotlib.pyplot as plt
+import seaborn as sns
+import numpy as np
+from scipy import stats
+
+sns.set()
+
+
+class DataAnalysis:
+    def __init__(self):
+        self.final_results = []
+
+    def analise_data(self):
+        """
+        Runs sequence of methods. Clears data, sort it and makes a plot.
+        :param col: int
+        :return: None
+        """
+        self.read_data_for_analysis()
+        self.split_data_for_analysis()
+        data = self.sort_data()
+        self.create_a_plot(data)
+
+    @staticmethod
+    def read_data_for_analysis():
+        """
+        Takes all files from folder 'results', reads it line by line and splits every each line into separate list
+        :return: None
+        """
+        for file in sorted(glob.glob('results/*')):
+            with open(file, 'r') as f:
+                return f.readlines()
+
+    def split_data_for_analysis(self):
```

Comment:
Tu metoda read_data... jest użyta poprawnie

Reply:
a no wlasnie, to wyzej napisalem